import_export/admin.py

```diff
@@ -577,6 +575,8 @@ class ExportActionModelAdmin(ExportActionMixin, admin.ModelAdmin):
     admin action.
     """
 
+    actions = admin.ModelAdmin.actions + [export_admin_action]
```

Comment:
I think `actions = admin.ModelAdmin.actions + [ExportActionMixin.export_admin_action]` should fix your broken tests.